File: sample-v7/src/main/java/io/erva/sample/base/cell/GammaCell.java
package io.erva.sample.base.cell;

import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import io.erva.celladapter.Layout;
import io.erva.celladapter.v7.Cell;
import io.erva.sample.BaseCell;
import io.erva.sample.R;
import io.erva.sample.base.model.GammaModel;

@Layout(R.layout.item_base_gamma)
public class GammaCell extends BaseCell<GammaModel, Cell.Listener<GammaModel>> {

    @BindView(R.id.tv_gamma) TextView textView;

    public GammaCell(View view) {
        super(view);
    }

    @Override
    protected void bindView() {
        textView.setText(getItem().getGamma());
    }
}

File: sample-kotlin/src/main/kotlin/io/erva/sample/base/cell/GammaCell.kt
package io.erva.sample.base.cell

import android.view.View
import io.erva.celladapter.Cell
import io.erva.celladapter.Layout
import io.erva.sample.R
import io.erva.sample.base.model.GammaModel
import kotlinx.android.synthetic.main.item_base_gamma.view.*

@Layout(R.layout.item_base_gamma)
class GammaCell(view: View) : Cell<GammaModel, Cell.Listener<GammaModel>>(view) {

    override fun bindView() {
        view.tv_gamma.text = item().gamma
    }
}